config/database.yml

```diff
@@ -0,0 +1,21 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: localhost
+  username: blog2
+  password: uiop7890
```

Comment:
- [ ] [OPTIONAL] It is not wise to upload real credentials to github. I suggest using environmental variables. For example the `<%= Rails.application.credentials.db_username %>`

Reply:
@Cathella, Thanks for the suggestions. I will keep in mind and try to apply that. :lock: